docs/guide/controller-plugin.md

```diff
@@ -68,11 +68,21 @@ Robot windows are implemented in HTML and provide the following features:
 The equivalent controller functions are `wb_robot_wwi_receive_text` and `wb_robot_wwi_send_text`.
 3. They are web-ready and could be used to display robot windows on web pages.
 
-A simple example of an HTML robot window is provided in the `robots/thymio/thymio2.wbt` sample simulation and demonstrates:
-
-- How to display sensor information in the robot window.
-- How to send user interface events (like mouse clicks) from the robot window to the controller program.
-- How to change the title of the robot window from the controller program.
+A simple example of an HTML robot window is given in [`projects/samples/howto/custom_robot_window_simple`](samples-howto.md#custom_robot_window_simple-wbt) and it demonstrates how to establish a two-way communication between a robot window and Python controller.
+To create a similar robot window for your project follow these steps:
+1. In your project's root create a file in the following path `plugins/robot_windows/<robot window name>/<robot window name>.html`.
+2. The file is a typical HTML document that can contain JavaScript and CSS.
+However, in addition to the standard JavaScript library, a `webots` object is injected that exposes an interface to allow communication with a robot controller.
+Therefore, you can check usage examples of `webots.window("<robot window name>").receive` and `webots.window("<robot window name>").send` in [`projects/samples/howto/custom_robot_window_simple/plugins/robot_windows/custom_robot_window_simple/custom_robot_window_simple.html`](https://github.com/cyberbotics/webots/tree/master/projects/samples/howto/custom_robot_window_simple/plugins/robot_windows/custom_robot_window_simple/custom_robot_window_simple.html).
+3. The robot window needs to be registered in the robot's node.
+Find your robot in the scene tree, select a `window` field, click select and choose `<robot window name>`.
+If you double click on the robot, your `<robot window name>` will open.
+4. To send and receive data inside the controller you have to use `wb_robot_wwi_receive_text` and `wb_robot_wwi_send_text`.
+A very simple Python example is given in [`projects/samples/howto/custom_robot_window_simple/controllers/custom_robot_window_simple/custom_robot_window_simple.py`](https://github.com/cyberbotics/webots/tree/master/projects/samples/howto/custom_robot_window_simple/controllers/custom_robot_window_simple/custom_robot_window_simple.py).
+
+An example of a reusable HTML robot window is given in [`projects/samples/howto/custom\_robot\_window`](samples-howto.md#custom_robot_window-wbt).
+While in the previous example the HTML robot window interacts directly with the controller, in this example the robot window exchanges data with a controller implemented in `plugins/robot_windows/custom_robot_window/custom_robot_window.c`.
+Therefore, the robot window in this case can be reused in different simulations.
```

Comment:
In the resulting documentation page, there is no empty line between this sentence and the next one.

Reply:
![image](https://user-images.githubusercontent.com/2135826/104444285-dc565480-5597-11eb-97ec-439c5754392b.png)
It looks good to me. Do we refer to the same thing?

Reply:
I would have expected that the `The HTML robot windows..` part would be in a different paragraph so that it is clear that is applies to both the robot windows.

Reply:
I added an empty line explicitly.

However, those were two different paragraphs according to markdown. GitHub renders them as two different paragraphs:
![image](https://user-images.githubusercontent.com/2135826/104445122-08bea080-5599-11eb-911a-9bfb1df4741e.png)